backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/bpartner/JsonRequestConsolidateService.java

```diff
@@ -77,6 +78,30 @@ public void consolidateWithIdentifier(@NonNull final JsonRequestLocationUpsert l
 		}
 	}
 
+	public static void consolidateWithOrg(@NonNull final JsonRequestBPartnerUpsertItem requestItem, @Nullable final String orgCode)
+	{
+		if (Check.isBlank(orgCode))
+		{
+			//nothing to consolidate
+			return;
+		}
+
+		final String requestItemOrgCode = requestItem.getBpartnerComposite().getOrgCode();
+
+		if (Check.isBlank(requestItemOrgCode))
+		{
+			requestItem.getBpartnerComposite().setOrgCode(orgCode);
+			return;
+		}
+
+		if (!orgCode.equals(requestItemOrgCode))
```

Comment:
ok, so we say that one can't just re-org a bpartner like this.
given the effort that we had to put into the change-org-prozess i think that is fair enough..

Reply:
pls, use `else if` instead of `if`... would help code static analyzers too.

Reply:
also wondering why this method is not going to JsonRequestBPartnerUpsertItem class?